src/main/java/com/github/johrstrom/listener/PrometheusServer.java

```diff
@@ -161,8 +162,9 @@ public synchronized void start() throws IOException {
         server.start();      
     }
     
-    public synchronized void stop() {
-    	server.stop(delay);
+    public synchronized void stop() throws InterruptedException {
+        TimeUnit.SECONDS.sleep(delay);
+    	server.stop(0);
```

Comment:
I haven't looked deeper, but I wonder if the problem should be fixed inside of the stop() method. It looks like non-zero stop() is broken by the way you fixed this. I think you should at least leave a comment explaining why fixing the server.stop() method is not preferred.

Reply:
I found the issue you created and read [the doc you linked](https://docs.oracle.com/javase/8/docs/jre/api/net/httpserver/spec/com/sun/net/httpserver/HttpServer.html#stop-int-). I would support this change with at least a more descriptive change in the CHANGELOG.  👍